normandy/recipes/api/views.py

```diff
@@ -156,6 +156,14 @@ def disable(self, request, pk=None):
         return Response(status=status.HTTP_204_NO_CONTENT)
 
 
+class RecipeHistoryViewSet(viewsets.ReadOnlyModelViewSet):
+    queryset = Version.objects.all()
```

Comment:
Is it possible to filter this queryset to only versions for Recipes? Or am I wrong in thinking that this might match versions that aren't for recipes?